src/renderer/index.js

```diff
@@ -23,8 +23,10 @@ const { autoUpdater } = remote.require('electron-updater');
 autoUpdater.logger = remote.require("electron-log");
 
 window.addEventListener('contextmenu', event => {
```

Comment:
The check you added gets rid of the error that was showing up, but now it just never reaches the `contextMenu()` line. This entire piece can just be removed

Reply:
You can remove it. It currently doesn't do anything. `contextMenu` isn't imported into that file, which is why the error is showing up. The actual context menu is handled in a different place